### PR TITLE
chore(release): update dependencies

### PR DESCRIPTION
**Pull request by paologallinaharbur**
Updating go to 1.18 and several dependencies

### CODE REVIEW

File: .github/workflows/nightly.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  GO_VERSION: '1.17'
+  GO_VERSION: '1.18'
```
Comment (gsanchezgavier) [inline .github/workflows/nightly.yaml:12]: too soon to 1.19?

**Review by gsanchezgavier — APPROVED** [review]
LGTM

File: .github/workflows/nightly.yaml (hunk 1 above)
Comment (paologallinaharbur) [reply .github/workflows/nightly.yaml:12]: yes 😄

Comment (paologallinaharbur) [reply .github/workflows/nightly.yaml:12]: I mean.. no, but lets first have everything for once at the same version

**Pull request merged**